examples/spice.mzLib.txt

```diff
@@ -0,0 +1,649 @@
+<mzSpecLib 1.0>
+MS:1003188|library name=spice
+MS:1003190|library version=1.0
+MS:1001017|release date=7/7/2023 7:06:42 PM
+MS:1003203|constituent spectrum file=mzspec:USI000000:spice_ph/cal_10ng.raw
+<AttributeSet Spectrum=all>
+MS:1000008|ionization type=MS:1000073|electrosprary ionization
+MS:1000465|scan polarity=MS:1000130|positive scan
+<Spectrum=1>
+MS:1003061|library spectrum name=JWH-250-5OH
+MS:1003057|scan number=795
+MS:1000894|retention time=3.53813
+MS:1003208|experimental precursor monoisotopic m/z =352.19000

```

Comment:
There should not be whitespace between term name and `=` sign

Reply:
Addressed